### PR TITLE
Add some tests to thingatp and hyperspec functionality

### PR DESCRIPTION
**Pull request by Sasanidas**

### CODE REVIEW

File: modes/lisp-mode/hyperspec.lisp [hunk 1]
```diff
@@ -0,0 +1,1006 @@
+(in-package :lem-lisp-mode/internal)
```
Comment (cxxxr) [inline modes/lisp-mode/hyperspec.lisp:1]: This might be a good idea to separate the packages. e.g. `lem-lisp-mode/hyperspec`

**Review by cxxxr — COMMENTED** [review]
Finally, hyperspec can be referenced! Thanks!

**Comment by cxxxr** [comment]
There are areas where the test has failed 👀

**Comment by Sasanidas** [comment]
> There are areas where the CI has failed eyes

Yeah, seems like it's the egg chicken problem, I'm using a command in thingatp and the tests load thingatp before the extensions where the command is :face_with_spiral_eyes:

**Comment by cxxxr** [comment]
> Yeah, seems like it's the egg chicken problem, I'm using a command in thingatp and the tests load thingatp before the extensions where the command is 😵‍💫

What if it load it after language-mode.lisp in lem.asd?

**Comment by Sasanidas** [comment]
> > Yeah, seems like it's the egg chicken problem, I'm using a command in thingatp and the tests load thingatp before the extensions where the command is face_with_spiral_eyes
> 
> What if it load it after language-mode.lisp in lem.asd?

Right! make sense :+1:

**Comment by Sasanidas** [comment]
Now are the tests that are failing because of the symbol

**Comment by cxxxr** [comment]
Oh, find-definitions is a symbol of the lem/language-mode package, not lem package.

**Comment by cxxxr** [comment]
Thank you so much!

**Pull request merged**